dandi/tests/expected_bids_validator_xs_write.log

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Please place this file and the other `*.log` file under `dandi/tests/data` (possibly in a dedicated subdirectory).